test/e2e/data/infrastructure-vsphere/v1.8/topology/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../bases/cluster-template-topology.yaml
+  - ../bases/cluster-resource-set.yaml
+patchesStrategicMerge:
+  - ../bases/cluster-resource-set-label.yaml
+  - ../bases/cluster-network-CIDR.yaml
+  - ../bases/cluster-resource-set-csi-insecure.yaml
+patches:
+  - target:
+      kind: VSphereMachineTemplate
+    path: ../bases/remove-storage-policy.yaml
```

Comment:
Do you know why we need this patch? If yes, can we add a comment here & on main?

Reply:
This removes the requirement for the test to have a working storage policy, while it is part of the published yamls so it could get defined.

xref where this removal was introduced: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/pull/1686/files

Reply:
Given that we have a working storage policy. Should we drop this patch in a follow-up PR?

Reply:
Let's learn the "why" the hard way :-)